apps/storefront/next.config.js

```diff
@@ -49,16 +49,20 @@ module.exports = withBundleAnalyzer({
         source: "/checkout/(.*)",
         headers: [{ key: "x-frame-options", value: "ALLOWALL" }],
       },
+      /**
+       * TODO. Ensure this is valid approach. Best if it can be limited to Saleor,
+       *   so maybe it should use NEXT_PUBLIC_API_URI?
+       */
+      {
+        source: "/dashboard/(.*)",
+        headers: [{ key: "content-security-policy", value: "frame-ancestors *" }],
+      },
```

Comment:
This page must be loaded in dashboard, we can lock specific dashboard in header?